torch/_export/serde/schema.py

```diff
@@ -154,15 +148,23 @@ class GraphSignature:
     user_inputs: List[str]
     user_outputs: List[str]
     buffers_to_mutate: Dict[str, str]
+    backward_signature: Optional[BackwardSignature]
+
+
+@dataclass
+class CallSpec:
     in_spec: str
     out_spec: str
-    backward_signature: Optional[BackwardSignature]
 
 
 @dataclass
 class GraphModule:
     graph: Graph
-    buffers: Dict[str, TensorMeta]
-    parameters: Dict[str, TensorMeta]
-    metadata: Dict[str, str]
+    opset_version: Dict[str, int]
```

Comment:
My naive approach is to serials a dict directly. It will be a namespace to op set version mapping. Example: {“aten”: 3, “custom_namespace”: 4}. We can change this in the future if needed.

Reply:
OK, so in any case, no mapping of op names (also str) to versions, yeah? That mapping will exist somewhere else in a table / code?